Class/ClassHelper.psm1

```diff
@@ -0,0 +1,44 @@
+function LogOutput {         
+    [CmdletBinding()]
+    param($msg)
+    $timestamp = (Get-Date).ToUniversalTime()
+    $output = "$timestamp [INFO]:: $msg"    
+    if ($VerbosePreference -ne "SilentlyContinue") {
```

Comment:
Good point. I believe this is leftover from my attempts to get the module to see that the -Verbose parameter was set. I'll remove it.